transpiler/languages/c_plus_plus.py

```diff
@@ -0,0 +1,42 @@
+code = {
+    # Output
+    'print': {
+        'entities': ['to_print'],
+        'code': 'cout << {to_print};'
+    },
+
+    # TODO: data type to use in looping, by default it's int.
+    'print_elements': {
+        'entities': ['to_print'],
+        'code': '''
+            for(const int &element : {to_print})
+                cout << element << endl;            
```

Comment:
`code` should be:
```c++
for (const auto& element : {to_print})
    cout << element << ", ";
cout << "\b\b \n";
```
The last line overwrites the trailing comma with a space and goes to a new line.